`b611` test case is broken

The `bsc.bugs/bluespec_inc/611` test case (which requires setting `DO_INTERNAL_CHECKS=1` to run) is currently broken:

```
$ make localcheck
cleaning /home/rscott/Documents/Hacking/Haskell/bsc/testsuite/bsc.bugs/bluespec_inc/b611
MAKEFLAGS= BSCTEST=1 BSC=/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/../inst/bin/bsc BSC_OPTIONS="" BSDIR=/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/../inst/lib DUMPBO=/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/../inst/bin/dumpbo BSC2BSV=/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/../inst/bin/bsc2bsv BSV2BSC=/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/../inst/bin/bsv2bsc VCDCHECK=/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/../inst/bin/vcdcheck SHOWRULES=/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/../inst/bin/showrules BLUESPECDIR=/home/rscott/Documents/Hacking/Haskell/bsc/inst/lib BSC_VERILOG_SIM=iverilog TEST_CONFIG_DIR=/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/config BLUETCL=/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/../inst/bin/bluetcl OSTYPE=Linux LC_ALL=C SYSTEMC_INC= SYSTEMC_LIB= PATH="/home/rscott/Documents/Hacking/Haskell/bsc/inst/lib/../bin:/home/rscott/.ghcup/tmp/ghcup-ghc-8.10.7:/home/rscott/.opam/4.14.0-coq-8.15.2/bin:/home/rscott/.ghcup/bin:/home/rscott/.cargo/bin:/home/rscott/.poetry/bin:/home/rscott/.elan/bin:/home/rscott/Software/bsc-2022.01/bin:/home/rscott/.cabal/bin:/opt/cabal/3.4/bin:/home/rscott/.idris2/bin:/opt/ghc/8.10.4/bin:/home/rscott/Software/armv7m-linux-musleabi-cross/bin:/opt/spark/bin:/opt/spark/sbin:/home/rscott/Software/what4-solvers:/home/rscott/.local/bin:/home/rscott/bin:/home/rscott/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/home/rscott/.local/bin:/home/rscott/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin"  runtest --tool "" --objdir . --status  *.exp
WARNING: Couldn't find tool init file
Test run by rscott on Fri Jan  6 09:46:02 2023
Native configuration is x86_64-pc-linux-gnu

                ===  tests ===

Schedule of variations:
    unix

Running target unix
Using /usr/share/dejagnu/baseboards/unix.exp as board description file for target.
Using /usr/share/dejagnu/config/unix.exp as generic interface file for target.
Using ./../../../config/unix.exp as tool-and-target-specific interface file.
testconfig dir is: /home/rscott/Documents/Hacking/Haskell/bsc/testsuite/config
Sourcing: /home/rscott/Documents/Hacking/Haskell/bsc/testsuite/config/verilog.tcl
testconfig dir is: /home/rscott/Documents/Hacking/Haskell/bsc/testsuite/config
Sourcing: /home/rscott/Documents/Hacking/Haskell/bsc/testsuite/config/bluetcl.tcl
Running ./b611.exp ...
ERROR: tcl error sourcing ./b611.exp.
ERROR: extra characters after close-quote
    while executing
"set status [exec_with_log "bsc2bsv"$"
    (procedure "bsc2bsv" line 14)
    invoked from within
"bsc2bsv $source"
    (procedure "run_bsc2bsv" line 11)
    invoked from within
"run_bsc2bsv Bug611.bs"
    (file "./b611.exp" line 1)
    invoked from within
"source ./b611.exp"
    ("uplevel" body line 1)
    invoked from within
"uplevel #0 source ./b611.exp"
    invoked from within
"catch "uplevel #0 source $test_file_name""

       === Test Distribution Summary === 
   run_bsc2bsv                              --     1
Total:                                      --     1

                === Timing by Category ===

  WALLCLOCK sec        CPU sec      SYSTEM sec   #calls  CATEGORY
     0.0 (100%)      0.0 (100%)      0.0 (100%)       0  TOTAL

                ===  Summary ===

make: *** [/home/rscott/Documents/Hacking/Haskell/bsc/testsuite/suitemake.mk:116: localcheck] Error 2
```

The culprit is a syntax error here:

https://github.com/B-Lang-org/bsc/blob/de3a575604d1418d29a441454fb0667b5415958b/testsuite/config/unix.exp#L1080

There should be a space between `"bsc2bsv"` and `$cmd`. This was not caught by CI since the CI does not enable `DO_INTERNAL_CHECKS`—see #530.

--- a/testsuite/config/unix.exp
+++ b/testsuite/config/unix.exp
@@ -1077,7 +1077,7 @@ proc bsc2bsv {source} {
     set output [make_bsc2bsv_output_name $source]
     set cmd "$bsc2bsv $source >& $output"
     verbose "Executing: $cmd" 4
-    set status [exec_with_log "bsc2bsv"$cmd 2]
+    set status [exec_with_log "bsc2bsv" $cmd 2]
 
     cd $here
     return [expr $status == 0]

